### PR TITLE
Maximelehericy add circle info

### LINKED ISSUE #10799: Document the Circles app in the user manual
At the moment the Circles app is documented nowhere in Nextcloud Manuals.

### PR DESCRIPTION
**Pull request by maximelehericy**
### ☑️ Resolves

* Fix #10799

### 🖼️ Screenshots

Added a block at the end of https://docs.nextcloud.com/server/latest/user_manual/en/groupware/contacts.html#

For some reason, I tried to add a screenshot, but it is not well displayed... (first time I am doing this though)

### CODE REVIEW

**Comment by maximelehericy** [comment]
@hamza221 @ChristophWurst would you review here ? I've running a POC at the moment and this information is missing for the customer... I'd better like to provide it within the official doc :)

**Review by ChristophWurst — COMMENTED** [review]
Change looks good. @ArtificialOwl please check the contents :)

File: user_manual/groupware/contacts.rst [hunk 1]
```diff
@@ -122,8 +122,65 @@ to create new address books, simply by specifying an address books name:
...
+
+In addition to admin permissions, an owner can transfer the circle ownership to another member of the circle. There can be only one single owner per circle.
+
+Add members to a circle
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Users, groups, or other circles can be added as members to a circle.
+For a group or a circle, the role applies to all members of the group or circle.
+
+**Be careful when adding other circles as member, as doing this too much can have an heavy impact on the overall performance of the system.**
```
Comment (tflidd) [inline user_manual/groupware/contacts.rst:181]: Does this also apply to groups?

File: user_manual/groupware/contacts.rst [hunk 2]
```diff
@@ -122,8 +122,65 @@ to create new address books, simply by specifying an address books name:
...
+.. figure:: ../images/circle.png
+	:alt: Circle in the Contacts app left menu
+
+Create a circle
+~~~~~~~~~~~~~~~
+
+In the left menu, click on the + next to Circles.
+Set a circle name
+Landing on the circle configuration screen, you can:
+- add users to your circle
```
Comment (ArtificialOwl) [inline user_manual/groupware/contacts.rst:147]: I would write "add _members_ to your circle" as a member can be a local account, a remote mail address, another circle, ...

File: user_manual/groupware/contacts.rst [hunk 3]
```diff
@@ -122,8 +122,65 @@ to create new address books, simply by specifying an address books name:
...
 
 See :doc:`index` for more details about syncing your address books
 with iOS, macOS, Thunderbird and other CardDAV clients.
+
+
+Circles
+-------
+
+Informal collaboration takes place within organizations: an event to organize for a few weeks, a short ideation session between members from differents entities, workshops, a place to joke around and support team building, or simply in very organic organizations where formal structure is kept to a minimum.
+
```
Comment (ArtificialOwl) [inline user_manual/groupware/contacts.rst:135]: "aggregate of _accounts_" ?

File: user_manual/groupware/contacts.rst [hunk 4]
```diff
@@ -122,8 +122,65 @@ to create new address books, simply by specifying an address books name:
...
+In addition to moderator permissions, an admin can configure circle options.
+
+**Owner**  
+
+In addition to admin permissions, an owner can transfer the circle ownership to another member of the circle. There can be only one single owner per circle.
+
+Add members to a circle
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Users, groups, or other circles can be added as members to a circle.
```
Comment (ArtificialOwl) [inline user_manual/groupware/contacts.rst:178]: - replace `Users` with `Local accounts`
- add the fact that a mail address can be used to add a member with no local account to a Circle

File: user_manual/groupware/contacts.rst (hunk 1 above)
Comment (ArtificialOwl) [inline user_manual/groupware/contacts.rst:181]: I am not even sure about the heavy impact on the overall performance of the system. 
If this is a the case, this is an issue: inheritance should be cached and should not affect the performance

File: user_manual/groupware/contacts.rst (hunk 3 above)
Comment (maximelehericy) [reply user_manual/groupware/contacts.rst:135]: ?

**Comment by maximelehericy** [comment]
good for you @ArtificialOwl

**Review by ArtificialOwl — APPROVED** [review]

File: user_manual/groupware/contacts.rst [hunk 5]
```diff
@@ -122,8 +122,63 @@ to create new address books, simply by specifying an address books name:
...
+
+
+Circles
+-------
+
+Informal collaboration takes place within organizations: an event to organize for a few weeks, a short ideation session between members from differents entities, workshops, a place to joke around and support team building, or simply in very organic organizations where formal structure is kept to a minimum.
+
+For all these reasons, Nextcloud supports Circles, a feature embedded in the Contacts app, where every user is able to create its own circle, a user-defined aggregate of accounts. Circles can be used later on to share files and folders, added to Talk conversations, like a regular group.
+
+.. figure:: ../images/circle.png
```
Comment (skjnldsv) [inline user_manual/groupware/contacts.rst:138]: This breaks building.
The circle image is missing

**Comment by nickvergessen** [comment]
> For some reason, I tried to add a screenshot, but it is not well displayed... (first time I am doing this though)

Well maybe this time would have been better.

```
Transcript written on Nextcloud_User_Manual.log.
Latexmk: Index file 'Nextcloud_User_Manual.idx' was written
Latexmk: Missing input file '{images/circle}.png' (or dependence on it) from following:
  'LaTeX Warning: File `{images/circle}.png' not found on input line 2801.'
Latexmk: Missing input file '{images/circle}.png' (or dependence on it) from following:
  'LaTeX Warning: File `{images/circle}.png' not found on input line 2801.'
Collected error summary (may duplicate other messages):
  pdflatex: Command for 'pdflatex' gave return code 1
      Refer to 'Nextcloud_User_Manual.log' for details
```

File: user_manual/groupware/contacts.rst (hunk 5 above)
Comment (skjnldsv) [reply user_manual/groupware/contacts.rst:138]: @maximelehericy what image were you planning to add?

Comment (skjnldsv) [reply user_manual/groupware/contacts.rst:138]: https://github.com/nextcloud/documentation/pull/10979

Comment (nickvergessen) [reply user_manual/groupware/contacts.rst:138]:
```suggestion
.. figure:: ./images/circle.png
```

**Pull request merged**